spec/features/authentication_pages_spec.rb

```diff
@@ -7,8 +7,12 @@
   describe 'authorization' do
     before { visit signin_path }
 
-    it { should have_content('Sign in') }
     it { should have_title('Sign in') }
+    it { should have_content('Sign in') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.